x-pack/plugins/cases/public/components/configure_cases/connectors.test.tsx

```diff
@@ -42,7 +46,7 @@ describe('Connectors', () => {
 
   beforeEach(() => {
     jest.clearAllMocks();
-    appMockRender = createAppMockRenderer();
+    appMockRender = createAppMockRenderer({ permissions: noConnectorsCasePermission() });
```

Comment:
nit: Do we need the `{ permissions: noConnectorsCasePermission() }`?

Reply:
Here it isn't necessary. I am deleting it.